infrastructure/src/main/java/org/corfudb/infrastructure/logreplication/replication/send/logreader/StreamsLogEntryReader.java

```diff
@@ -82,23 +84,32 @@ public class StreamsLogEntryReader implements LogEntryReader {
 
     public StreamsLogEntryReader(CorfuRuntime runtime, LogReplicationConfig config) {
         runtime.parseConfigurationString(runtime.getLayoutServers().get(0)).connect();
+        this.config = config;
```

Comment:
Update the class-level comment adding that the streams to replicate will get refreshed 1) on the start of every log entry sync and 2)whenever a new stream to replicate is discovered.